iOS/immersive-reader-sdk/immersive-reader-sdk/ImmersiveReaderViewController.swift

```diff
@@ -4,139 +4,133 @@
 import UIKit
 import WebKit
 
-public class ImmersiveReaderViewController: UIViewController, WKUIDelegate, WKNavigationDelegate {
+public class ImmersiveReaderViewController: UIViewController {
     
-    let tokenToSend: String
-    let subdomainToSend: String
-    let contentToSend: Content
-    let optionsToSend: Options?
-    let onSuccessImmersiveReader: (() -> Void)?
-    let onFailureImmersiveReader: ((_ error: Error) -> Void)?
-    let onTimeout: ((_ timeoutValue: TimeInterval) -> Void)?
-    let onError: ((_ error: String) -> Void)?
-    
-    let startTime = Date().timeIntervalSince1970*1000
-    public var src = "https://learningtools.onenote.com/learningtoolsapp/cognitive/reader"
-    var webView: WKWebView!
-    var timer: Timer!
-    var timeoutValue: TimeInterval!
-    
-    public init(tokenToPass: String, subdomainToPass: String, contentToPass: Content, optionsToPass: Options?, onSuccessImmersiveReader: @escaping () -> Void, onFailureImmersiveReader: @escaping (_ status: Error) -> Void, onTimeout: @escaping (_ timeoutValue: TimeInterval) -> Void, onError: @escaping (_ error: String) -> Void) {
-        self.tokenToSend = tokenToPass
-        self.subdomainToSend = subdomainToPass
-        self.contentToSend = contentToPass
-        self.optionsToSend = optionsToPass
-        self.onSuccessImmersiveReader = onSuccessImmersiveReader
-        self.onFailureImmersiveReader = onFailureImmersiveReader
-        self.onTimeout = onTimeout
-        self.onError = onError
-        super.init(nibName: nil, bundle: nil)
-    }
-    
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
+    private let accessToken: String
+    private let subdomain: String
+    private let content: Content
+    private let completionHandler: ((_ error: Error?) -> Void)?
+    private let startTime: TimeInterval
+
+    private var src = "https://learningtools.onenote.com/learningtoolsapp/cognitive/reader"
```

Comment:
this may need to change considering @rwallerms 's PR 
https://github.com/microsoft/immersive-reader-sdk/pull/123

@rwallerms  any comment?

Reply:
Not using src now. Modified the code to use JS library directly so resolving.